.github/workflows/build-test-manylinux-aarch64.yaml

```diff
@@ -26,9 +26,19 @@ jobs:
           buildx-fallback: true
       - name: Extract wheel from docker image
         run: |
+          # load the docker image containing the semgrep python wheel
           docker load --input /tmp/image.tar
-          docker run --platform linux/arm64 --rm ${{ steps.build-semgrep-wheel.outputs.imageid }} cat cli/dist.zip > /tmp/dist.zip
+
+          # create a new docker container using the image we just loaded
+          # note: `docker create` simply prepares the container filesystem -- nothing is executed!
+          CONTAINER_ID=$(docker create ${{ steps.build-semgrep-wheel.outputs.imageid }})
+
+          # use `docker export` to extract the python wheel zipfile out of the container
+          docker export $CONTAINER_ID | tar xv semgrep/cli/dist.zip
```

Comment:
could you add a comment explaining what the alternative used to be and why it was wrong? think it would help to read this code in the future (and also its not clear to me why the previous method wasn't writing all of `stdout`)